test/jest/Alias/Alias.test.ts

```diff
@@ -1,13 +1,54 @@
-import { substituteAliases, parseAliasDeclaration } from "../../../src/Alias";
+import { substituteAliases, parseAliasDeclaration, clearAliases } from "../../../src/Alias";
 describe("substituteAliases Tests", () => {
-  it("Should gracefully handle recursive local aliases", () => {
+  beforeEach(() => {
+    clearAliases();
+  });
+  it("Should gracefully handle recursive local aliases I", () => {
     parseAliasDeclaration("recursiveAlias=b");
     parseAliasDeclaration("b=c");
     parseAliasDeclaration("c=d");
     parseAliasDeclaration("d=recursiveAlias");
 
     const result = substituteAliases("recursiveAlias");
-    expect(result).toEqual("d");
+    expect(result).toEqual("recursiveAlias");
+  });
+
+  it("Should gracefully handle recursive local aliases II", () => {
+    parseAliasDeclaration("recursiveAlias=recursiveAlias");
+    const result = substituteAliases("recursiveAlias");
+    expect(result).toEqual("recursiveAlias");
+  });
+
+  it("Should gracefully handle recursive local aliases III", () => {
+    parseAliasDeclaration('recursiveAlias="recursiveAlias"');
+    const result = substituteAliases("recursiveAlias");
+    expect(result).toEqual("recursiveAlias");
+  });
+
+  it("Should gracefully handle recursive local aliases IV", () => {
+    parseAliasDeclaration('recursiveAlias="recursiveAlias -l"');
+    const result = substituteAliases("recursiveAlias");
+    expect(result).toEqual("recursiveAlias -l");
+  });
+
+  it("Should not substitute quoted commands I", () => {
+    parseAliasDeclaration("a=b");
+    const result = substituteAliases('"a"');
+    expect(result).toEqual('"a"');
+  });
+
+  it("Should not substitute quoted commands II", () => {
+    parseAliasDeclaration("a=b");
+    const result = substituteAliases("'a'");
+    expect(result).toEqual("'a'");
+  });
+
+  it("Should not substitute quoted commands III", () => {
+    parseAliasDeclaration("a=b");
+    parseAliasDeclaration("b='c'");
+    parseAliasDeclaration("c=d");
+    const result = substituteAliases("a");
+    //expect(result).toEqual("'c'"); // Currently FAILS
```

Comment:
You can write this test as `skip` or `failing` (they have different semantics; `failing` is *expected* to fail, `skip` is just not run since the result is ignored).

Reply:
there is also xtest or xit, which I find funny, but I'd use test.skip here.